airbyte-integrations/connectors/source-mysql/src/test-performance/java/io/airbyte/integrations/source/mysql/FillMySqlTestDbScriptTest.java

```diff
@@ -90,30 +84,32 @@ protected String getTestFieldType() {
     return TEST_DB_FIELD_TYPE;
   }
 
-  @Override
-  protected String getNameSpace() {
-    return container.getDatabaseName();
-  }
-
   /**
-   * Creates all tables and insert data described in the registered data type tests.
-   *
-   * @throws Exception might raise exception if configuration goes wrong or tables creation/insert
-   *         scripts failed.
+   * The test added test data to a new DB and check results. 1. Set DB creds in static variables above
+   * 2. Set desired number for streams, coolumns and records 3. Run the test
    */
-  private void setupDatabaseInternal() throws Exception {
-    final Database database = setupDatabase();
+  @Test
+  @Disabled
+  public void addTestDataToDbAndCheckTheResult() throws Exception {
```

Comment:
This method for MySQL is almost the same as those for Postgres and MSSQL. If a `numberOfBatches` is added to this method, I think all three databases can share the same method to insert the data, and this method can be lifted to the upstream class. In this way, the three classes should be more straightforward to read.

Reply:
Updated, thanks!